publications.md

```diff
@@ -13,27 +13,6 @@ If you are looking for a paper to cite, this is the one you want:
 
 Do you have a paper about PMIx or that uses PMIx in an interesting way and would like it listed here? Just let us know!
 
-Papers
-------
-
-2019
-====
-
- - Matteo Turilli, Andre Merzky, Thomas Naughton, Wael R. Elwasif, Shantenu Jha. Characterizing the Performance of Executing Many-Tasks on Summit. Third Annual Workshop on Emerging Parallel and Distributed Runtime Systems and Middleware (IPDRM), Nov. 2019 \[[Paper](https://conferences.computer.org/sc19w/2019/#!/toc/11)\]
- - Geoffroy Vallee, Carlos Eduardo Arango Gutierrez, and Cedric Clerget. On-node resource manager for containerized HPC workloads. IEEE/ACM Workshop on Containers and New Orchestration Paradigms for Isolated Environments in HPC (CANOPIE-HPC), Nov. 2019 \[[Paper](https://www.canopie-hpc.org/wp-content/uploads/2019/12/PMIx-Canopie-HPC-2019.pdf)\]
- - Dong Zhong, Aurelien Bouteiller, Xi Luo, and George Bosilca. Runtime level failure detection and propagation. In Proceedings of the 26th European MPI Users’ Group Meeting, EuroMPI 2019, pages 9:1–9:10, New York, NY, USA, 2019. ACM. \[[Paper](https://doi.org/10.1145/3343211.3343225)\]
- - Artem Y. Polyakov, Boris I. Karasev, Joshua Hursey, Joshua Ladd, Mikhail Brinskii and Elena Shipunova. A performance analysis and optimization of PMIx-based HPC software stacks. In Proceedings of the 26th European MPI Users’ Group Meeting, EuroMPI 2019, pages 9:1–9:10, New York, NY, USA, 2019. ACM. \[[Paper](https://doi.org/10.1145/3343211.3343220)\]
-
-2018
-====
-
- - Ralph H. Castain, David Solt, Joshua Hursey, and Aurelien Bouteiller. PMIx: Process management for exascale environments. In Parallel Computing, vol. 79, pages 9-29, Nov. 2018 \[[Paper](https://doi.org/10.1016/j.parco.2018.08.002)\]
- - Artem Y. Polyakov, Joshua Ladd, Elena Shipunova, and Boris I. Karasev. A Scalable PMIx Database (poster) EuroMPI 2018 Conference (EuroMPI’18) \[[Abstract](https://eurompi2018.bsc.es/sites/default/files/uploaded/dstore_empi2018.pdf)\] \[[Paper](https://eurompi2018.bsc.es/sites/default/files/uploaded/Polyakov_PMIx_EuroMPI2018_poster.pdf)\]
-
-2017
-====
-
- - Ralph H. Castain, David Solt, Joshua Hursey, and Aurelien Bouteiller. PMIx: Process management for exascale environments. In Proceedings of the 24th European MPI Users’ Group Meeting, EuroMPI ’17, pages 14:1–14:10, New York, NY, USA, 2017. ACM.
```

Comment:
I'm not sure what the right answer is for publications.  As one route, I could see putting basically any paper associated with PMIx (implementation or standard) on this page.  As another route, I could see only having standard-related stuff listed here.  Either way, I wonder if at least the EuroMPI and PARCO papers should remain.  They are kind of the foundational papers, right?

Same thing for presentations.  It's kind of hard to tease apart standards from implementation talks since they were synonymous for so long.  Not sure what the best move is, but we should probably be consistent across both.

Thoughts?

Reply:
Agreed.  I actually should not have cut the Castain2017 paper, that was a mistake. I agree that having papers on standards page that relate to more fundamental elements is good.  

Same on presentations, but harder to make that distinction.   Maybe we leave all the existing presentations (with pointer somewhere to OpenPMIx Pubs page) and decide going forward where to place things.   It will probably be a little easier for that decision, example, WG and ASC presentations can go on Standard Pubs page, but talks that are actually using OpenPMIx would go there. 

I'll make change to this PR to not remove those 2 pubs.

Reply:
Note, I put the Paper at end (after Presentations) for two reasons.
 - 1. To avoid any confusion w/ OpenPMIx Pubs site (less so with color diffs now)
 - 2. Anticipate more Presentations/Slides for Standard page so bias to list those first

Feel free to rearrange as appropriate.  One other thought is to avoid the distinction of Presentations/Papers here and just use "YEAR" on Standard page.